GCRCatalogs/catalog_configs/desc_dc2_run2.2i_star_truth_summary_int_id.yaml

```diff
@@ -1,8 +1,25 @@
 #  Use ^/ to indicate file path relative to GCR root dir
 
-subclass_name: dc2_truth_parquet.DC2TruthParquetCatalog
-base_dir: ^/lsstdesc-public/dc2/run2.2i-dr6-v2/truth_variable
-filename_pattern: 'star_truth_summary_int_id.parquet$'
+subclass_name: composite.CompositeReader
```

Comment:
There should be something in the config name to indicate stats are include. Could be, e.g., `desc_dc2_run2.2i_star_truth_summary_with_stats_int_id.yaml` or `desc_dc2_run2.2i_star_truth_summary_int_id_with_addons.yaml`. I'm open to other suggestions.

Reply:
Right, again, we need the naming guideline yesterday.  Here we are not providing a "not with addon" option, and this catalog will not result in any confusion if people thought it doesn't have the addon columns... 

So I don't think we with those additions in the name.  But if you want to make a decision here, then I'll follow that decision.

Reply:
The reason I think this is important, even though it makes a long unwieldy name even longer, is that `truth_summary` tables have a well-defined structure https://docs.google.com/document/d/1PnFZqdMmHBk6ePEvZBdKVOt64vwQ1cse2SaSJTVx7GM/edit  which we've respected up to this point.

Reply:
Ok in the to-be-written naming guidelines we should also specify how to move forward when people can't reach an agreement 😂

I think we have different philosophies for what catalog names' main purpose is, so it might be difficult for us to agree here.

If we were to add some new columns to an existing catalog, I don't think we will need to change the catalog name, and I think the same logic applies here. Yes, the truth summary has a specific schema, but I don't see why we can't add a few new columns to it and say that those columns are only available for stars, not SNe. To me, changing the catalog name does not help the users, as in both cases they still need to read the release notes to see what's available.

But again, I think our debate here is not likely to be fruitful. Instead of trying to convince each other, one of us can make a decision. Since you are both the person who prepared the catalog and also the co-convener, I think it makes sense for you to make the decision here and I'll just follow it. 

If you want me to make the decision instead, I can do that too. Just let me know.

Reply:
Sorry, I didn't see your last post until now.   
Probably this whole pull request and #587 also have been superseded by events.

Reply:
This one is certainly superseded. We won't need composite catalog with the new proposal. Some of the #587 should be kept -- I'll comment there.